scripts/cc-cli/src/utils/era.ts

```diff
@@ -21,3 +21,27 @@ export async function timeTillEra(api: ApiPromise, era: number) {
 
   return timeTillTargetEra;
 }
+
+export async function checkEraIsInHistory(
+  era: number,
+  api: ApiPromise
+): Promise<boolean> {
+  const currentEra = (await api.query.staking.currentEra()).value.toNumber();
+  const historyDepth = api.consts.staking.historyDepth.toNumber();
+  return eraIsInHistory(era, historyDepth, currentEra);
+}
+
+export function eraIsInHistory(
+  era: number,
+  historyDepth: number,
+  currentEra: number
+): boolean {
+  // The oldest era in history is currentEra - historyDepth
+  // https://polkadot.js.org/docs/kusama/constants/#historydepth-u32
+  const oldestEraInHistory = currentEra - historyDepth;
```

Comment:
Yeah, it is odd. I was expecting to find some sort of method to get the ones in history in order to prevent duplicating the calculation, but did not. For the meantime I added a check to the command for ensuring `era` is a positive number when calling `distribute-rewards`.

Reply:
+1 for the check inside `parseOptions`. However that's unrelated to my comment. 

For example when `currentEra = 50` and `historyDepth = 84` then `oldestEraInHistory` will be `-34`.  It's odd but so far the comparison works so I'm marking this as resolved.